### PR TITLE
Add informations about custom caddy rules.

### PR DESCRIPTION
**Pull request by AiroPi**
Add some "documentation" on how to add custom caddy rules with the caddy community container.

### CODE REVIEW

File: community-containers/caddy/readme.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This container bundles caddy and auto-configures it for you. It also covers http
...
 - If you want to use this with https://github.com/nextcloud/all-in-one/tree/main/community-containers/stalwart, make sure that you point `mail.your-nc-domain.com` to your server using a cname record so that caddy can get a certificate automatically for stalwart.
 - After the container was started the first time, you should see a new `nextcloud-aio-caddy` folder and inside there an `allowed-countries.txt` file when you open the files app with the default `admin` user. In there you can adjust the allowed country codes for caddy by adding them to the first line, e.g. `IT FR` would allow access from italy and france. Private ip-ranges are always allowed. Additionally, in order to activate this config, you need to get an account at https://dev.maxmind.com/geoip/geolite2-free-geolocation-data and download the `GeoLite2-Country.mmdb` and upload it with this exact name into the `nextcloud-aio-caddy` folder. Afterwards restart all containers from the AIO interface and your new config should be active!
+- You can add your own Caddy configurations in `/data/caddy-imports/` inside the Caddy container (`docker exec -it nextcloud-aio-caddy bash`). These will be imported on container startup.
```
Comment (szaimen) [inline community-containers/caddy/readme.md:10]:
```suggestion
- You can add your own Caddy configurations in `/data/caddy-imports/` inside the Caddy container (`sudo docker exec -it nextcloud-aio-caddy bash`). These will be imported on container startup.
```

**Comment by szaimen** [comment]
Hey @AiroPi I just gave you write permissioins to the repo so that you can create branches and do more things here :)

**Comment by AiroPi** [comment]
> Hey @AiroPi I just sent you an invite for the repo and gave you write permissioins to the repo so that you can create branches and do more things here :)

Thank you for your confidence !

**Comment by szaimen** [comment]
BTW @AiroPi since you seem to like the community-container feature, maybe you are interested to add one more? See https://github.com/nextcloud/all-in-one/discussions?discussions_q=is%3Aopen+label%3A%22help+wanted%22 for some existing ideas :)

**Comment by AiroPi** [comment]
> BTW @AiroPi since you seem to like the community-container feature, maybe you are interested to add more? See https://github.com/nextcloud/all-in-one/discussions?discussions_q=is%3Aopen+label%3A%22help+wanted%22+sort%3Atop for some existing ideas :)

I'd be happy to do more. However, I won't have much time in the next few months. But as soon as I have a bit of time, I'll be able to tackle the following ones, if they haven't already been done by then:
- pgAdmin
- netdata
- eventually calcardbackup, but calendar is already backed up with borg, right ?

For the other ones, I don't know the apps and tools, or I don't need it at all.
I'd also look at adding HomeAssistant, which I plan to install! As well as Grafana. But I'll see in the meantime.

**Comment by szaimen** [comment]
Great to hear! And no worries, take your time! :)

**Pull request merged**